pages/index.jsx

```diff
@@ -14,7 +14,7 @@ function Home() {
       <section className="hero">
         <div className="container">
           <div className="text-wrapper">
-            <h1 data-cy="homePageTitle" className="title">
+            <h1 data-test-id="homePageTitle" className="title">
```

Comment:
Change the `data-cy` attribute from Cypress to `data-test-id` attribute from Playwright